test/bin/run_plugin.spec.js

```diff
@@ -189,5 +188,26 @@ describe('Run plugin CLI', function () {
                 }
             );
         });
+
+        it('should run Minimal Working Example on a project where only the user has access', function (done) {
+            gmeAuth.addUser('specialUser', 'special@access.com', 'specialUser', true, {overwrite: true})
+                .then(function () {
+                    return testFixture.importProject(storage, {
+                        projectSeed: './test/bin/run_plugin/project.webgmex',
+                        projectName: 'specialProject',
+                        branchName: 'master',
+                        gmeConfig: gmeConfig,
+                        username: 'specialUser',
+                        logger: logger
+                    });
+                })
+                .then(function () {
+                    return Q.nfcall(runPlugin.main, ['node', filename,
```

Comment:
You could check the commit history in the plugin results and see who made the commit.